src/index.html

```diff
@@ -11,7 +11,112 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form 
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application" 
+      method="post"
+      autocomplete="off"
+    >
+      <fieldset class="tabel">
+        <legend>Personal information</legend>
+        <label class="form__field">
+          Surname:
+          <input type="text" name="Surname">
+        </label>
+        <label class="form__field">
+          Name:
+          <input type="text" name="Name">
+        </label>
+        <label class="form__field">
+          How old are You?
+          <input 
+            type="number" 
+            name="Age" 
+            value="12" 
+            min="1" 
+            max="100"
+          >
+        </label>
+        <label class="form__field">
+          Full date of birth:
+          <input type="date" name="dateOfBirth">
+        </label>
+        <label class="form__field">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms" required>
```

Comment:
```suggestion
          <input 
            type="checkbox" 
            name="terms" 
            required
          >
```
Please fix all cases to pass the tests
![image](https://user-images.githubusercontent.com/96409563/212034049-73b1fe0f-d806-4f1e-8bba-e251b85be74a.png)